IPython/core/debugger.py

```diff
@@ -62,6 +63,7 @@ def BdbQuit_excepthook(et, ev, tb, excepthook=None):
     All other exceptions are processed using the `excepthook`
     parameter.
     """
+    warnings.warn("`BdbQuit_excepthook` is deprecated", DeprecationWarning)
```

Comment:
Nitpicking, but once we agree on  when to deprecate (5.1 or 6.0), I like to add `...deprecated since version X.y` otherwise it's painful for user to figure out when they can remove potential shims , and for us to figure out when we can delete the code.